imports/ui/components/login/client/login.tests.js

```diff
@@ -0,0 +1,94 @@
+import {
+    name as Login
+} from '../login';
+
+import 'angular-mocks';
+
+import {
+    Meteor
+} from 'meteor/meteor';
+
+import {
+    sinon
+} from 'meteor/practicalmeteor:sinon';
+
+describe('Login', function() {
+    spies.restoreAll();
+
+    beforeEach(function(done) {
+        window.module(Login);
+        done();
+    });
+
+    describe('controller', function() {
+        let controller;
+
+        beforeEach(function(done) {
+            inject(function($rootScope, $componentController) {
+                controller = $componentController(Login, {
+                    $scope: $rootScope.$new(true)
+                });
+            });
+            done();
+        });
+
+        it('should have credentials all empty by default', function(done) {
+            expect(controller.credentials).to.be.deep.equal({
+                email: '',
+                password: ''
+            });
+            done();
+        });
+
+        it('should have error empty by default', function(done) {
+            expect(controller.error).to.be.equal('');
+            done();
+        });
+
+        describe('login()', function() {
+            let validEmail = 'validEmail';
+            let validPassword = 'validPassword';
+
+            afterEach(function(done) {
+                if (spies.login)
+                    spies.login.restore();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'spies'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2729684798/issues/source?bid=3340041&fileBranchId=3346427#l54)